doc/howto/custom_field_type.md

```diff
@@ -160,6 +160,31 @@ class RelationFieldDefinitionMapper extends DecoratingFieldDefinitionMapper impl
 }
 ```
 
+#### Field Definition Input Mappers
+As of v1.0.4, an extra interface is available for mutation input type handling, `FieldDefinitionInputMapper`.
+It is used if the input for this field depends on the field definition. For instance, `ezmatrix`
+generates its own input types depending on the configured columns.  It defines an extra method, `mapToFieldValueInputType`, 
+that returns a GraphQL type for a Field Definition.
+
+Example:
+```
+class MyFieldDefinitionMapper extends BaseFieldDefinitionMapper implements FieldDefinitionMapper, FieldDefinitionInputMapper
+{
+    public function mapToFieldValueInputType(ContentType $contentType, FieldDefinition $fieldDefinition): ?string
+    {
+        // Forward compatibility with 2.x.
```

Comment:
I'd rather say either `// Forward compatibility with eZ Platform 3.x.` or `// Forward compatibility with version 2.x` of this bundle.
Because those versions are close with eZ Platform, it might be confusing.

Reply:
Fixed.